test/framework/easyblock.py

```diff
@@ -847,27 +847,28 @@ def test_fetch_patches(self):
 
         eb.fetch_patches()
         self.assertEqual(len(eb.patches), 2)
-        self.assertEqual(eb.patches[0]['name'], 'toy-0.0_typo.patch')
+        self.assertEqual(eb.patches[0]['name'], 'toy-0.0_fix-silly-typo-in-printf-statement.patch')
         self.assertFalse('level' in eb.patches[0])
 
         # reset
         eb.patches = []
 
+        toy_patch = 'toy-0.0_fix-silly-typo-in-printf-statement.patch'
```

Comment:
This one should be moved up I guess so `toy_patch` can also be used above in this test, I overlooked that.